development/nautobot_config.py

```diff
@@ -260,14 +260,15 @@
             "jobs": True,
             "models": {
                 "dcim": {
-                    "Site": True,
-                    "Rack": True,
-                    "Device": True,
+                    "Site": ["status__name", "region__name"],
```

Comment:
So this is going to drive a breaking change we need to keep that in mind with the release version number